content/docs/cucumber/api.md

```diff
@@ -1041,6 +1041,35 @@ class RunCucumberTest {
 ```
 The default option for `strict` is `false`.
 
+For example if you want to tell Cucumber to print code snippets for missing
+step definitions use the plugin `summary`, you can specify it like this:
```

Comment:
step definitions, you can use the `summary` plugin. It is specified like this ...